core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -62,8 +69,43 @@ public static function reset()
 	 * @param boolean $blnAsChunks If true a ChunkedText object is returned instead of the string
 	 *
 	 * @return string|ChunkedText The text with the replaced tags
+	 *
+	 * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0.
+	 *             Use the InsertTagParser service instead.
 	 */
 	public function replace($strBuffer, $blnCache=true, $blnAsChunks=false)
```

Comment:
If this will get merged into 4.13 we could probably remove `$blnAsChunks` again (it's not released yet) here and just keep it in the internal method.